addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -198,7 +205,7 @@ export class LinkPlugin extends Plugin {
                 title: _t("Button"),
                 description: _t("Add a button"),
                 categoryId: "navigation",
-                commandId: "openLinkTools",
+                commandId: "openLinkToolsButton",
```

Comment:
You should not need to create a new command for that. You should be able to use `commandParams`. If needed, you can adapt the `openLinkTools` command so the params are properly passed. I think it would be better than to create another command just for that.

Reply:
Yes indeed, thankss